l10n_th_personal_income_tax/views/account_move_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@id='header_right_group']" position="inside">
+                <field
+                    name="account_pit"
+                    attrs="{'invisible': [('move_type', '!=', 'in_invoice')]}"
+                />
+                <field
+                    name="pit_wht_cert_income_type"
```

Comment:
Should this set required, if account_pit is checked?
Note: I am not sure about this, please confirm with BA if user will know income type now.